crates/scheduler/src/call/store.rs

```diff
@@ -76,16 +76,16 @@ fn find<B: Board>(mut call: SchedulerCall<B, api::find::Sig>) {
     let mut memory = scheduler.applet.memory();
     let result = try {
         match scheduler.store.find(*key as usize) {
-            Ok(None) => Ok(false),
+            Ok(None) => false,
             Ok(Some(value)) => {
                 let len = value.len() as u32;
                 let ptr = memory.alloc(len, 1)?;
                 memory.get_mut(ptr, len)?.copy_from_slice(&value);
                 memory.get_mut(*ptr_ptr, 4)?.copy_from_slice(&ptr.to_le_bytes());
                 memory.get_mut(*len_ptr, 4)?.copy_from_slice(&len.to_le_bytes());
-                Ok(true)
+                true
             }
-            Err(e) => Err(convert(e)),
+            Err(e) => Err(convert(e))?,
```

Comment:
this branch could be removed with `match scheduler.store.find(*key as usize).map_err(convert)? {`